spyder_terminal/widgets/main_widget.py

```diff
@@ -79,13 +79,11 @@ def __init__(self, name, plugin, parent):
         self.server_ready = False
         self.font = None
         self.port = select_port(default_port=8071)
-        self.server_stdout = subprocess.PIPE
-        self.server_stderr = subprocess.PIPE
-        self.stdout_file = osp.join(os.getcwd(), 'spyder_terminal_out.log')
-        self.stderr_file = osp.join(os.getcwd(), 'spyder_terminal_err.log')
+        self.stdout_file = None
+        self.stderr_file = None
         if get_debug_level() > 0:
-            self.server_stdout = open(self.stdout_file, 'w')
-            self.server_stderr = open(self.stderr_file, 'w')
+            self.stdout_file = osp.join(os.getcwd(), 'spyder_terminal_out.log')
+            self.stderr_file = osp.join(os.getcwd(), 'spyder_terminal_err.log')
```

Comment:
What do you think about creating these files in our config directory and adding an entry to see them in Spyder Tools menu? I say that because Ryan is trying to do something similar in PR https://github.com/spyder-ide/spyder/pull/15629 and it's necessary for our installers.

In case it's something that you find useful, please open an issue about it so you can address it later (it's not something for this PR).

Reply:
I'm unable to do this because the way the menu is being created is not compliant with the API I have available outside Spyder. I think to do this we may need to do some changes in Spyder so the Application plugin has this menu available.